server/routes/utilMongoose.js

```diff
@@ -0,0 +1,132 @@
+var express = require('express');
+var router = express.Router();
+var crypto = require('crypto-random-string');
+var bcrypt = require('bcrypt');
+var mongoose = require('mongoose')
+var userSchema = require('../models/userSchema');
+var alumniSchema = require('../models/alumniSchema');
+
+const HASH_COST = 10;
+
+router.get('/getAlumni/', async (req, res, next) => {
```

Comment:
let's name this path `/alumni` so the `method` is indication of whether it's GET/POST

Reply:
Changed